!Command
command:
- anvi-migrate
positional: []
named: []
parent:
subcommands: []
usage: []
help_flag:
usage_flag:
version_flag:
help_text: "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.6/site-packages/anvio/utils.py\"\
  , line 3450, in check_h5py_module\n    import h5py\nModuleNotFoundError: No module\
  \ named 'h5py'\n\nDuring handling of the above exception, another exception occurred:\n\
  \nTraceback (most recent call last):\n  File \"/usr/local/bin/anvi-migrate\", line\
  \ 16, in <module>\n    from anvio.migrations import migration_scripts\n  File \"\
  /usr/local/lib/python3.6/site-packages/anvio/migrations/__init__.py\", line 24,\
  \ in <module>\n    spec.loader.exec_module(module)\n  File \"/usr/local/lib/python3.6/site-packages/anvio/migrations/contigs/v9_to_v10.py\"\
  , line 15, in <module>\n    utils.check_h5py_module()\n  File \"/usr/local/lib/python3.6/site-packages/anvio/utils.py\"\
  , line 3452, in check_h5py_module\n    raise ConfigError(\"Please install the Python\
  \ module `h5py` manually for this migration task to continue. \"\nanvio.errors.ConfigError:\
  \ \n\nConfig Error: Please install the Python module `h5py` manually for this migration\
  \ task to     \n              continue. The reason why the standard anvi'o installation\
  \ did not install module\n              is complicated, and really unimportant.\
  \ If you run `pip install h5py` in your   \n              Python virtual environmnet\
  \ for anvi'o, and try running the migration program    \n              again things\
  \ should be alright.                                                 \n\n\n"
generated_using:
- --help
